index.html

```diff
@@ -2411,17 +2411,22 @@ <h2>
         Installable web applications
       </h2>
       <p>
-        A common use case of a manifest is for a user agent to
-        <dfn data-local-lt="installing|installation" data-lt=
-        "installed">install</dfn> a web application; whereby the user agent
-        provides the end-user with a means of instantiating a new <a>top-level
-        browsing context</a> that has the manifest's members <a>applied</a> to
-        it. A web application that is installed is known as a <dfn data-export=
-        "">installed web application</dfn>. That is, the manifest's members, or
-        their defaults, are in effect on the <a>top-level browsing context</a>.
-        This distinguishes an installed web application from a traditional
-        bookmark, as opening a web page from a traditional bookmark will not
-        have the manifest's properties <a>applied</a> to it.
+          Any website is an <strong>installable web application</strong>. 
+        <p>
+        </p>
+        <p>
+        A user agent can provide a way for the end-user to <dfn data-local-lt="installing|installation" data-lt=
+        "installed">install</dfn> a web application on the end-user's device, allowing the
+        user to instantiate a new [=top-level browsing context=] with the
+        manifest's members [=applied=].
+        </p>
+        <p>Once a web application is [=installed=] it is known as a 
+        <dfn class="export">installed web application</dfn>: 
+        That is, the manifest's members, or their defaults,
+        are [=applied=] to the <a>top-level browsing context</a> of the web application. This
+        distinguishes an installed web application from a traditional bookmark,
+        as opening a web page from a traditional bookmark will not have the
+        manifest's properties <a>applied</a> to it.
```

Comment:
> Once a web application is installed it is known as a[sic] installed web application: That is, the manifest's members, or their defaults, are applied to the top level browsing context of the web application.

This could be refined to say that _when the app is launched_ the manifest's members are applied to the created browsing context?

Currently it reads to me like an installed web application is one whose manifest _has_ been applied to a top-level browsing context. But actually installing a web app and launching a web app may be two separate steps as described in the following note. A web application which has been added to an app launcher but not yet launched may still be considered "installed".